### ISSUE TITLE
Fern buffer does not stay open

### ISSUE THREAD

**Issue by j-hui** [lambdalisue/fern.vim#388]
When I switch away from a fern buffer, it disappears from the buffer list. Is there any way to keep it around? I'd like to keep "directory buffers" in my buffer list alongside regular file buffers.

I saw that https://github.com/lambdalisue/fern.vim/issues/75 deals with this issue, though in the opposite direction, so perhaps this is intentional.

**Comment by j-hui** [comment]
I realized it's because the viewer is setting the `nobuflisted` option [here](https://github.com/lambdalisue/fern.vim/blob/b6204ec4e04732d7b4ba73a3e4af48cba3576d67/autoload/fern/internal/viewer.vim#L49) and [here](https://github.com/lambdalisue/fern.vim/blob/b6204ec4e04732d7b4ba73a3e4af48cba3576d67/autoload/fern/internal/viewer.vim#L57). Is it possible to add an option to disable these?

**Comment by lambdalisue** [comment]
You can use `is_drawer()` function explained in `fern-develop.txt`.
See `:help fern-develop-helper.sync.is_drawer()` and context around that.

**Comment by j-hui** [comment]
FYI: I couldn't figure out a workaround within the parameters of the existing plugin, so I created [this fork](https://github.com/j-hui/fern.vim/tree/buflisted) to see what would happen if I made the `nobuflisted` option configurable.

In particular, drawers are still `nobuflisted`, but non-drawers are.

It seems to work fine in the regular case but it gets a little hairy once there are multiple open Fern buffers (including drawer + non-drawers) for the same directory. Also, opening a directory creates another Fern buffer.
